### ISSUE TITLE
comparing python and .NET objects, not possible by design?

### ISSUE THREAD

**Issue by denfromufa** [pythonnet/pythonnet#130]
```
>>> from System import Double
>>> Double(5.5)
<System.Double object at 0x02886550>
>>> Double(5.5)==5.5
False
```

http://sourceforge.net/p/pythonnet/bugs/2/

**Comment by denfromufa** [comment]
```
IronPython 2.7.7 (2.7.7.0) on .NET 4.0.30319.42000 (32-bit)
Type "help", "copyright", "credits" or "license" for more information.
>>> from System import Double
>>> Double(5.5)==float(5.5)
True
```

**Comment by denfromufa** [comment]
closed in favor of #626
